scripts/circle-ci/run-circle-tests.sh

```diff
@@ -38,10 +38,10 @@ fi
 
 case $CIRCLE_NODE_INDEX in
     0) ./gradlew $TEST_CONTAINER_ARGS check $CONTAINER_0_EXCLUDE_ARGS ;;
-    1) ./gradlew $TEST_CONTAINER_ARGS ${CONTAINER_1[@]} ;;
-    2) ./gradlew $TEST_CONTAINER_ARGS ${CONTAINER_2[@]} -x :atlasdb-ete-tests:longTest ;;
+    1) CASSANDRA_VERSION=2.2.8 ./gradlew $TEST_CONTAINER_ARGS ${CONTAINER_1[@]} ;;
```

Comment:
Do we want to add these? If so, we should add them to all gradlew commands so rebalancing doesn't become weird.